agent.go

```diff
@@ -541,9 +541,9 @@ func (s *sandbox) listenToUdevEvents() {
 			}
 
 			s.Unlock()
-		} else if strings.HasPrefix(uEv.DevPath, sysfsMemOnlinePath) {
+		} else if strings.HasPrefix("/sys"+uEv.DevPath, sysfsMemOnlinePath) {
```

Comment:
Two comments here:
- Please define a `var` or `const` for `/sys`, something like `var sysfsDir = "/sys"` that can be reused across the whole agent code.
- Use `filepath.Join()` if you want to concatenate `/sys` with the uevent `DevPath`.

Reply:
Yep - I'd consider using a `var` so we can make it testable - see #387.